lib/nm/source.rb

```diff
@@ -35,8 +36,8 @@ def render(template_name, locals = nil)
 
     private
 
-    def source_file_path(template_name)
-      Dir.glob(self.root.join("#{template_name}*")).first
+    def source_file_path(name)
+      Dir.glob(self.root.join(name.end_with?(@ext) ? name : "#{name}*#{@ext}")).first
```

Comment:
@kellyredding - Yeah this looks like what we discussed. I didn't know `end_with?` was a thing. Nice.